src/components/learner-credit-management/BudgetDetailAssignments.jsx

```diff
@@ -10,6 +10,81 @@ import BudgetAssignmentsTable from './BudgetAssignmentsTable';
 import AssignMoreCoursesEmptyStateMinimal from './AssignMoreCoursesEmptyStateMinimal';
 import { useBudgetContentAssignments, useBudgetId, useSubsidyAccessPolicy } from './data';
 
+const BudgetDetailAssignmentsHeader = ({
+  isRetired,
+}) => {
+  const assignedHeadingRef = useRef();
+  const navigate = useNavigate();
+  const location = useLocation();
+
+  const { state: locationState } = location;
+
+  useEffect(() => {
```

Comment:
The original `useEffect` for doing the `.scrollIntoView` for the "Assigned" table is still implemented in `BudgetDetailAssignments`, but is essentially a no-op because the `assignedHeadingRef` in `BudgetDetailAssignments` is not passed to an element anymore.

The new `assignedHeadingRef` within `BudgetDetailAssignmentsHeader` is the only ref still used.

I would recommend removing the `useEffect` that intends to do the same thing from `BudgetDetailAssignments` since it's not really doing anything anymore.